src/py-opentimelineio/opentimelineio-bindings/otio_anyVector.cpp

```diff
@@ -19,6 +19,14 @@ void otio_any_vector_bindings(py::module m) {
     
     py::class_<AnyVectorProxy>(m, "AnyVector")
         .def(py::init<>())
+        .def(py::init([](const py::iterable &it) {
+            any a;
+            py_to_any(it, &a);
+            AnyVector v = safely_cast_any_vector_any(a);
+            auto proxy = new AnyVectorProxy;
+            proxy->fetch_any_vector().swap(*v.get_or_create_mutation_stamp()->any_vector);
+            return proxy;
+        }))
```

Comment:
is this copying the data?

Reply:
If by data you mean everything (like a deep copy in Python), no. If you mean the top level list/vector, yes. See https://github.com/AcademySoftwareFoundation/OpenTimelineIO/pull/1574/files#r1165693980.

What should be the behavior?